src/frontend/EdFi.DataManagementService.Frontend.AspNetCore/Modules/MetadataEndpointModule.cs

```diff
@@ -15,10 +15,18 @@ namespace EdFi.DataManagementService.Frontend.AspNetCore.Modules;
 
 public partial class MetadataEndpointModule : IEndpointModule
 {
+    private sealed record SpecificationSection(string name, string prefix);
+
     [GeneratedRegex(@"specifications\/(?<section>[^-]+)-spec.json?")]
     private static partial Regex PathExpression();
 
-    private readonly string[] Sections = ["resources", "descriptors", "discovery"];
+    private readonly SpecificationSection[] Sections =
+    [
+        new SpecificationSection("resources", string.Empty),
+        new SpecificationSection("descriptors", string.Empty),
+        new SpecificationSection("discovery", "Other")
```

Comment:
This was an interesting discovery: look at [api.ed-fi.org/v7.1/api/metadata/](https://api.ed-fi.org/v7.1/api/metadata/) and you'll see that some items have a prefix and some do not. Those that _do not_ have a prefix are parsed by the bulk loader. And that was breaking with the Discovery API. Fix: set a prefix! Now the bulk loader ignores the Discovery specification.